tests/component/views/create-dataset/CreateDataset.spec.tsx

```diff
@@ -0,0 +1,29 @@
+// import DatasetCreate from '../../../../src/sections/create-dataset/CreateDatasetContext'
+import { CreateDatasetForm } from '../../../../src/sections/create-dataset/CreateDatasetFactory'
+import { mount } from 'cypress/react18'
+
+describe('Form component', () => {
+  it('renders the Create Dataset page and its contents', () => {
+    mount(<CreateDatasetForm />)
```

Comment:
Can you replace mount, by the custom mount Cypress command that we're using in the rest of component tests?

`cy.customMount`